internal/sources/cloudsqlmysql/cloud_sql_mysql.go

```diff
@@ -53,9 +53,9 @@ type Config struct {
 	Project  string         `yaml:"project" validate:"required"`
 	Region   string         `yaml:"region" validate:"required"`
 	Instance string         `yaml:"instance" validate:"required"`
-	IPType   sources.IPType `yaml:"ipType"`
-	User     string         `yaml:"user" validate:"required"`
-	Password string         `yaml:"password" validate:"required"`
+	IPType   sources.IPType `yaml:"ipType" validate:"required"`
```

Comment:
I believe we hard code this to public if not set. We shouldn't update this because then it's a breaking change to require it in the yaml.